index.html

```diff
@@ -162,6 +162,7 @@ <h3><dfn>DatasetCoreFactory</dfn> interface</h3>
       <dfn>dataset</dfn>
     </p>
     <p>Returns a new dataset and imports all quads, if given.</p>
+    <div class="note">The given sequence must not be mutated during import.</div>
```

Comment:
Following @rubensworks suggestion, what about something like:
```suggestion
    <div class="note">The given sequence must be treated as immutable (do not `.pop()`, `.splice()`, `.shift()`, etc).</div>
```